BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -137,6 +137,12 @@ public float getSWLWOffset
         [KSPField]
         public bool radarLOAL = false;
 
+        [KSPField]
+        public bool hasIOG = false;
+
+        [KSPField]
+        public bool hasBI = false;
```

Comment:
These could use more descriptive tags, like "hasIntertialGuidance" and 'basicInertial' so configs remain somewhat human-readable.